### PR TITLE
delete criteria filter code + modify tests

### PR DESCRIPTION
**Pull request by basseche**
Lib used by :

GridSuite/dynamic-mapping-server/pom.xml
GridSuite/dynamic-simulation-server/pom.xml
GridSuite/network-modification/pom.xml
GridSuite/filter-server/pom.xml
GridSuite/loadflow-server/pom.xml

### CODE REVIEW

File: src/test/java/org/gridsuite/filter/utils/FiltersUtilsTest.java
Comment (thangqp) [inline src/test/java/org/gridsuite/filter/utils/FiltersUtilsTest.java:1]: Normally, all tests in this file are already covered by other tests in expert filter.. you can try to remove this file and check code coverage whether having a change

Comment (basseche) [reply src/test/java/org/gridsuite/filter/utils/FiltersUtilsTest.java:1]: ok

File: src/test/java/org/gridsuite/filter/utils/FiltersUtilsTest.java [hunk 1]
```diff
@@ -25,8 +23,15 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
-import java.util.*;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.UUID;
```
Comment (Mathieu-Deharbe) [inline src/test/java/org/gridsuite/filter/utils/FiltersUtilsTest.java:31]: Why replace `import org.gridsuite.filter.expertfilter.expertrule.StringExpertRule;` with `import org.gridsuite.filter.expertfilter.expertrule.*;` ?

and just after  replace `import java.util.*; `with
```
import java.util.ArrayList;
import java.util.Date;
import java.util.List;
import java.util.Set;
import java.util.TreeSet;
import java.util.UUID;
```
?

**Review by Mathieu-Deharbe — COMMENTED** [review]
Here : 

```
public enum FilterType {
    SCRIPT,
    CRITERIA,
    IDENTIFIER_LIST,
    EXPERT,
}
```
You can probably remove CRITERIA,.

I tested without it and didn't encounter any problem.

**Review by thangqp — APPROVED** [review]

File: src/test/java/org/gridsuite/filter/utils/FiltersUtilsTest.java (hunk 1 above)
Comment (basseche) [reply src/test/java/org/gridsuite/filter/utils/FiltersUtilsTest.java]: modified

**Review by Mathieu-Deharbe — APPROVED** [review]

**Pull request merged**